samples/exemplarchat/src/main/kotlin/com/squareup/chat/ChatService.kt

```diff
@@ -12,15 +15,18 @@ import misk.web.MiskWebModule
 
 fun main(args: Array<String>) {
   val environment = Environment.fromEnvironmentVariable()
-  val config = MiskConfig.load<ChatConfig>("chat", environment.name)
+  val env = Env(environment.name)
+  val deployment = Deployment(name = "exemplarchat", isLocalDevelopment = true)
+
+  val config = MiskConfig.load<ChatConfig>("chat", env)
 
   MiskApplication(
       MiskRealServiceModule(),
       MiskWebModule(config.web),
       RealEventRouterModule(environment),
```

Comment:
Next phase is updating all Misk modules to use `Deployment` or `Env`. Once this is updated, won't need both `env` and `environment` variables above.